src/renderer/components/deposit/withdraw/Withdraw.tsx

```diff
@@ -313,42 +336,74 @@ export const Withdraw: React.FC<Props> = ({
     () =>
       FP.pipe(
         withdrawFeesRD,
-        RD.map((fees) => [{ asset: AssetRuneNative, amount: Helper.sumWithdrawFees(fees) }])
+        RD.map(({ rune: runeFees, asset: assetFees }) => [
+          { asset: AssetRuneNative, amount: Helper.sumWithdrawFees(runeFees) },
+          { asset: assetFees.asset, amount: assetFees.amount }
+        ])
       ),
     [withdrawFeesRD]
   )
 
   const reloadFeesHandler = useCallback(() => {
-    reloadFees(AssetRuneNative)
-  }, [reloadFees])
-
-  const disabledForm = useMemo(() => withdrawPercent <= 0 || disabled, [withdrawPercent, disabled])
+    reloadFees(asset)
+  }, [reloadFees, asset])
+
+  // Load fees by every `onMount`
+  useEffect(() => {
+    reloadFees(asset)
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [])
```

Comment:
just to clarify and keep in mind. right now this works as we use flag `destroyInactiveTabPane` at the [Deposit.tsx component](https://github.com/thorchain/asgardex-electron/blob/develop/src/renderer/components/deposit/Deposit.tsx#L153). reloading deposit fees is working now for every switch between `Deposit` and `Withdraw` screens for the same reason. 
we just should keep in mind that if we decide (for any reason) to remove that `destroyInactiveTabPane` tab we have to reload fees for every chain's switching

Reply:
yes. correct. 

> Since adding [asset] has no effect, I think it's ok to stay with current approach.
yep. right now there is no difference. lets keep only `onMount` effect but also keep in mind such scenario